### PR TITLE
Miscellaneous docs fixes

### PR DESCRIPTION
**Pull request by mbolivar**
I noticed some issues in the existing docs while thinking about where to add documentation for west, mostly in the getting started pages. Posting fixes for these separately.

### CODE REVIEW

File: doc/getting_started/getting_started.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ by Zephyr:
  * ``make``: Supported on UNIX-like platforms (Linux, macOS).
  * ``ninja``: Supported on all platforms.
 
-Most of the examples in the Zephyr documentation use `ninja` as a build tool
+Most of the examples in the Zephyr documentation use `ninja` as a build tool,
```
Comment (dbkinder) [inline doc/getting_started/getting_started.rst]: while you're fixing things, use double backticks for
 
    ``ninja``

File: doc/getting_started/getting_started.rst [hunk 2]
```diff
@@ -70,27 +70,19 @@ Use the following procedures to create a new development environment.
 Building and Running an Application
 ***********************************
 
-Using the 'Hello World' sample application as a base model, the following
```
Comment (dbkinder) [inline doc/getting_started/getting_started.rst]: I'm not fond of having a heading with no content after them except for another heading.  Can we put this first paragraph back in?

**Review by dbkinder — CHANGES_REQUESTED** [review]
little tweaks to your changes

File: doc/getting_started/getting_started.rst (hunk 2 above)
Comment (mbolivar) [reply doc/getting_started/getting_started.rst]: Hm, I don't think we ought to put it back in as-is. As I tried to explain in the commit log, this paragraph contains factual errors. I'll try to put something better in here.

**Review by dbkinder — APPROVED** [review]
Thanks.

**Review by nashif — APPROVED** [review]

**Pull request merged**